src/bin/tikv-importer.rs

```diff
@@ -132,8 +132,8 @@ fn setup_config(matches: &ArgMatches) -> TiKvConfig {
         fatal!("invalid configuration: {:?}", e);
     }
     info!(
-        "using config: {}",
-        serde_json::to_string_pretty(&config).unwrap()
+        "using config";
+        "info" => serde_json::to_string_pretty(&config).unwrap()
```

Comment:
Maybe better to not using pretty since it is not useful anymore? @huachaohuang PTAL

Reply:
I keep it here since this info is mostly been reading when we need to check the config in the log file and seems pretty string will be more friendly.

Reply:
I think it is not necessary to use a structured log here at all, these logs are only printed on startup.

Reply:
So I'd like to keep the origin log in all config info.

Reply:
@AndreMouche @huachaohuang No matter you use it as a field or in a message, these pretty print will be useless but introduce `\n` strings in the output.. Because anyway the logger will JSON serialize it..